pallets/author-mapping/src/mock.rs

```diff
@@ -109,7 +109,7 @@ impl frame_system::Config for Runtime {
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
 }
 parameter_types! {
-	pub const ExistentialDeposit: u128 = 1;
+	pub const ExistentialDeposit: u128 = 0;
```

Comment:
Why do we set this to 0 in some cases (mocks) and not others? `ethereum-xcm`, for example, still uses a non-zero ED.

Reply:
Yes I didn't modify the mocks that weren't failing, but I agree that this is a nice opportunity to modify all of them with ED of 0 for more consistency.